### ISSUE TITLE
GUI: Improper handling of column data

### ISSUE THREAD

**Issue by nist-aharvey** [usnistgov/REFPROP-issues#70]
1) Choose specified state points.
2) Type in a few rows of pressures.
3) For temperature, use the functionality where you right-click on the heading to specify values.  Choose T starting at 293 K, incrementing by 0, through the last row where you typed in pressures. Hit OK.
4) This calculates values as expected.
5) Decide you want more digits displayed on an output quantity, like Entropy, by right-clicking on that heading and changing the digits.  Then hit OK.
6) It pastes values of 293 into the Entropy column!

I think the problem occurs because it “remembers” the specify values information that you did previously, which is convenient in some circumstances.  But in this case, trying to change formatting on a calculated column, the Specify Values stuff isn’t even visible to the user, so there is no way the user can keep it from pasting in the stuff that is unwanted.  I think the solution is to make sure the “Specify values” stuff is never used in cases where it is not displayed to the user.
An alternative solution might be for it to only re-use Specify Values information if you have right-clicked on the same column where you specified the values previously, but that might be harder to implement.

**Comment by EricLemmon** [comment]
I'm not quite sure how you were able to make this happen (where the "Specify Values" section is not visible).  The right hand side (labeled "Specify Values") of the "Column Options" menu (obtained by right clicking on a header in a table of calculated points) is always visible for "Calculate/Specified State Points" tables.  It is not visible for fixed tables created by the first three options under the Calculate menu.  Even if the right hand side of the "Column Options" menu contained information from a previous paste in a different table, it will not paste values into a fixed table.  

Thus, the only possible way (I think) for this to happen would be to right-click on the header in a non-fixed table, fill in the right hand side of the menu that appears, use this to fill in cells in the table, repeat in a second column to calculate state points, then right click again on a header, enter the number of digits you would like in the "Digits" box, but forget to remove the data values from the previous entries in the "Specify Values" section.  This will overwrite the old calculated values with the new values being pasted in.  But the right hand side is always visible and thus you have the option to remove those values first.

For others reading this, below is a picture of what the menu looks like.

![image](https://user-images.githubusercontent.com/42392984/55270921-bb116080-526a-11e9-9370-60a1bf002586.png)

**Comment by nist-aharvey** [comment]
I played around a little more, and it looks like for the bug to happen the column checked in Step 5 of my description above must be a column that is not valid as an input quantity, like Cp0.  So I do the following steps:
0) In Properties, add Cp0 to the list.
1) Specified State Points.
2) Manually enter a few pressures (I did 5 rows)
3) Fill in the temperature with 273, from row 1 to row 5, increment of 0.
4) Calculations look fine.
5) Now right-click the Cp0 column (to change the decimal).  The Specify Values part is not visible (I will paste in a picture of the window), so if I try to change the number of digits it pastes 273 into the Cp0 column and there's nothing I can do about it.

So there must be logic that suppresses the Specify Values part of the window for some columns in Specified State Points (maybe when it is not a valid input column?), so I think the same logic should ensure that nothing gets pasted in those cases.

![image](https://user-images.githubusercontent.com/45049145/55332623-fff7eb80-5452-11e9-9367-f5611d4062f1.png)
